contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_checkstyle.py

```diff
@@ -111,9 +112,9 @@ def create_py2_failing_target(self):
             "a/python/fail_py2.py",
             contents=dedent(
                 """
-                x=2+3
-                print x+7
-                """
+                         x=2+3
+                         print x+7
+                       """
```

Comment:
These look wrong, along with the others in this file. The original is preferred:

```
"""
x=2+3
print x+7
"""
```

Feel free to ignore, though, if these still work as this module is being actively killed.

Reply:
Fixed